packages/wp-source/src/libraries/api.ts

```diff
@@ -1,5 +1,4 @@
-import fetch from "cross-fetch";
-
+import { fetch } from "frontity";
 class Api {
```

Comment:
Missing empty line between `import` and `class` :)

Reply:
Oops!